### PR TITLE
[ENGG-1554] chore: added typography tokens

### PR DESCRIPTION
**Pull request by wrongsahil**
Closes issue: 

## ✅ Checklist:

- [ ] Make sure linting and unit tests pass.
- [ ] No install/build warnings introduced.
- [ ] Verified UI in browser.
- [ ] For UI changes, added/updated analytics events (if applicable).
- [ ] For changes in extension's code, manually tested in Chrome and Firefox.
- [ ] For changes in extension's code, both MV2 and MV3 are covered.
- [ ] Added/updated unit tests for this change.
- [ ] Raised pull request to update corresponding documentation (if already exists).

### CODE REVIEW

File: app/src/lib/design-system-v2/helpers/theme.css [hunk 1]
```diff
@@ -80,5 +80,32 @@
   --requestly-color-white: #ffffff;
   --requestly-color-white-t-20: #ffffff33;
   --requestly-color-black: #0e0e0e;
-  --requestly-color-black-t-70: 000000b2;
+  --requestly-color-black-t-70: #000000b2;
```
Comment (rohanmathur91) [inline app/src/lib/design-system-v2/helpers/theme.css:83]: What does  `t` represents in ` --requestly-color-black-t-70`

File: app/src/lib/design-system-v2/helpers/theme.css [hunk 2]
```diff
@@ -80,5 +80,32 @@
...
+  --requestly-font-size-5xl: 3rem;
+  --requestly-font-size-6xl: 3.75rem;
+  --requestly-font-line-height-xs: 1rem;
+  --requestly-font-line-height-sm: 1.25rem;
+  --requestly-font-line-height-base: 1.5rem;
+  --requestly-font-line-height-lg: 1.75rem;
+  --requestly-font-line-height-xl: 1.75rem;
+  --requestly-font-line-height-2xl: 2rem;
+  --requestly-font-line-height-3xl: 2.25rem;
+  --requestly-font-line-height-4xl: 2.5rem;
```
Comment (rohanmathur91) [inline app/src/lib/design-system-v2/helpers/theme.css:110]: We should add a doc comment for there pixel equivalent values e.g
```css
/* 12px */
 --requestly-font-size-xs: 0.75rem;
 ```

**Review by rohanmathur91 — APPROVED** [review]

File: app/src/lib/design-system-v2/helpers/theme.css (hunk 1 above)
Comment (wrongsahil) [reply app/src/lib/design-system-v2/helpers/theme.css:83]: Transparency

File: app/src/lib/design-system-v2/helpers/theme.css (hunk 2 above)
Comment (wrongsahil) [reply app/src/lib/design-system-v2/helpers/theme.css:110]: theme.css is just for reference. In future we'll make this dynamic depending upon user selected font size

Comment (wrongsahil) [reply app/src/lib/design-system-v2/helpers/theme.css:110]: We'll not need the px value as all the values defined in designs will be using these values only

**Pull request merged**